lib/masks/falco_gen_azimuthal_phase_mask.m

```diff
@@ -111,14 +141,14 @@
             Nsteps = inputs.Nsteps;
 
             % Generate 1D phase profile 
-            Nsamps1D = 100*N; 
+            Nsamps1D = 1000*N; 
```

Comment:
If this 1000 value is the same as in line 151, then it would be better to make it a variable so no one accidentally changes one but not the other.

Reply:
changed this to a variable as suggested.